### ISSUE TITLE
electrum: `blockchain.transaction.get` verbose transaction is not supported

### ISSUE THREAD

**Issue by tiero** [Blockstream/esplora#423]
Any reason why the verbose param is not supported? any plan to add it anytime soon? @philippem @shesek

**Comment by shesek** [comment]
See https://github.com/Blockstream/electrs/pull/36 (and in particular https://github.com/Blockstream/electrs/pull/36#issuecomment-936850568).

Can you tell me more about your use case?

**Comment by tiero** [comment]
It was to avoid fetching the whole raw block and parsing it on the client side to only get the unix timestamp of a specific tx.

I see your answer and we are ok-ish with this approach, we maybe try to lazy load timestamps in client-side apps, they are not that crucial after all.

**Comment by shesek** [comment]
If that helps, you can get the block height (but not block timestamp) via `blockchain.transaction.get_merkle`.

I could be okay with supporting a minimal version of `blockchain.transaction.get` with `verbose=true` that does not attempt to be compatible, where it only returns the fields that *cannot* be extracted from the raw transaction alone (these related to block inclusion - `blockhash`, `blocktime` and `confirmations`).

However, returning a different json format under the same method name might be confusing for developers, and it may lead to weird behaviors for clients that expects the full json format (they should fail early and relatively cleanly if `verbose=true` returns an error, while returning a partial json could result in weirder more unexpected behaviors/bugs).

Open to hearing more thoughts on this :)

/cc @greenaddress

**Comment by tiero** [comment]
Yes a lot of clients already check for that specific error message. I think it's totally fine to not have it so we do not need to diverge from other ElectrumX servers, we can figure out other ways tho.
